Lab10/closedunidirectionallinkedlist.py

```diff
@@ -0,0 +1,198 @@
+from device import Device
+
+
+class Node:
+    def __init__(self, data: Device = None) -> None:
+        self.data: Device = data
+        self.next: Node = None
+
+
+class ClosedUnidirectionalLinkedList:
+    def __init__(self) -> None:
+        self.head: Node = Node(None)
+        self.tail: Node = Node(None)
+        self.head.next = self.tail
+        self.tail.next = self.head
+
+    # add -> add some element type Device to list
```

Comment:
what you did here with comments is fine, but you might want to read about [docstrings](https://peps.python.org/pep-0257/) as well.